dataland-frontend/src/components/forms/parts/fields/FormListFormField.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="form-field">
-    <UploadFormHeader :label="label" :description="description" :is-required="required" />
+    <UploadFormHeader :label="label" v-if="description" :description="description" :is-required="required" />
```

Comment:
Now if description is empty no description is shown? Why did we not need this before or was it just forgotten back then?

Reply:
We have components in which the <UploadFormHeader is part of the component and we don't want to display it if it is not needed. Previously, there was probably no case in which we didn't want to display the <UploadFormHeader

Reply:
ok